gradio/themes/base.py

```diff
@@ -224,65 +224,65 @@ def expand_shortcut(shortcut, mode="color", prefix=None):
     def set(
```

Comment:
We should add `font` and `font_mono` as arguments to `set` so that people can load a font from the hub and then be able to change the font